compose.yaml

```diff
@@ -9,4 +9,4 @@ services:
   ldap:
     image: ghcr.io/zakodium/ldap-with-users:1
     ports:
-      - 127.0.0.1:389:389
+      - 389:389
```

Comment:
this is not recommended as it opens the server to all network interfaces

Reply:
This is also puzzling me... My first instinct was to change the port mapping because of wrong port in the error message, but I also don't see why it fixes the issue.